README.md

```diff
@@ -17,3 +17,11 @@ GitLab resources, like users, groups and projects.
 ## Contributing
 
 Check out the [CONTRIBUTING.md](/CONTRIBUTING.md) guide for tips on how to contribute and develop the provider.
+
+## Community
+
+You are welcome to join our [Discord Server](https://discord.gg/CCc6Pe5EYy) or the monthly community hours:
+
+- [Google Calendar signup](https://calendar.google.com/event?action=TEMPLATE&tmeid=MnZyN2owdmYxaDdiMWdjNGdoMzQ3bWM0dDlfMjAyMjA0MjdUMTQzMDAwWiBjXzh0bjczaDNjYW5mam1zY2pwamJqNjFpNDVrQGc&tmsrc=c_8tn73h3canfjmscjpjbj61i45k%40group.calendar.google.com&scp=ALL )
+- [Recordings](https://www.youtube.com/playlist?list=PL05JrBw4t0KrMncKb6v9S0wB_liBky39P)
+- [Notes](https://docs.google.com/document/d/1-rgmylTUlO2lPRmbmGk0aB-mktg8nYu0NxOkFw9bo_M/edit#)
```

Comment:
I think we should remove the notes and share them ad-hoc during the meeting, given that we may share reserved information from time to time (GitLab SIRT already pinged me because the doc is open) 
```suggestion
```

Reply:
As the community hours are public, I'd rather add a disclaimer at the top of the docs to assure we are SAFE and to remind ourselves about the nature of the info we can share.

I'd prefer to keep the notes easily accessible and public as people might prefer to skim through them, instead of watching a recording. Moreover, once it gets shared, it might be shared outside of our control, so not linking to it here just obfuscates the risk.

Reply:
I updated the docs. Please, check.

Reply:
I'm taking the liberty to resolve this. Let me know if any additional change is required or we need another discussion about this :)